Makefile

```diff
@@ -84,5 +87,13 @@ njs-unit-test: ## Run unit tests for the njs httpmatches module
 		node:18 \
 		/bin/bash -c "npm install && npm test && npm run clean"
 
+.PHONY: generate-njs-yaml
+generate-njs-yaml:
+	kubectl create configmap njs-modules --from-file=$(NJS_DIR)/httpmatches.js --dry-run=client --output=yaml > $(MANIFEST_DIR)/njs-modules.yaml
```

Comment:
I suggest adding something to our dev guide and the source file itself, so that a developer is aware they need to regenerate something

Reply:
Will do, I haven't completed all the documentation changes yet (it's called out in the To-do list in the description) On that point:

> Shall we also update the main README?

same comment 😄